Documentation/Blazorise.Docs/Components/ComponentApiDocsProperty.razor

```diff
@@ -0,0 +1,60 @@
+﻿@using Blazorise.Generator.Features.ApiDocsDtos
+
+@if (filteredApiDocsForComponents.Any())
+{
+    <Heading Margin="Margin.Is2.FromTop.Is3.FromBottom" Size="HeadingSize.Is3">
+        @(PropertyType ==PropertyTypes.Events
+             ? "Events"
+            : "Parameters" )
+    </Heading>
+}
+
+@foreach(var apiDocsForComponent in filteredApiDocsForComponents)
+{
+    <DocsAttributes ShowDefaults="@(PropertyType != PropertyTypes.Events)"
+        Name="@(PropertyType == PropertyTypes.Events ? "Event": "Parameter")"
+        Title="@(MultipleComponentTypes ? apiDocsForComponent.TypeName : null)">
+        @foreach(var property in (PropertyType == PropertyTypes.Events? apiDocsForComponent.Events:apiDocsForComponent.Parameters).OrderBy(x => x.Name))
+        {
+            <DocsAttributesItem @key="@property.Name" Name="@property.Name" Type="@property.TypeName"
+                                Default="@property.DefaultValueString">
+                <Paragraph Margin="@(property.IsBlazoriseEnum ? null : Margin.Is0.FromBottom)">
+                    @((MarkupString)property.Summary)
+                </Paragraph>
+
+                @if (property.IsBlazoriseEnum)
+                {
+                    var enumNames = Enum.GetNames(property.Type);
+
+                    <Paragraph Margin="Margin.Is0.FromBottom">
+                        <Text>Possible values:</Text>
+                        @for (int i = 0; i < enumNames.Length; i++)
+                        {
+                            int i1 = i;
```

Comment:
I'd be more descriptive, localI, localIdx, etc... almost never numbers help with describing in my personal opinion